azure-pipelines.yml

```diff
@@ -198,23 +197,20 @@ stages:
     - bash: |
         set -e
         mne sys_info -pd
-        mne sys_info -pd | grep "qtpy: .*{PySide6=.*}$"
```

Comment:
It's really useful to have `qtpy` in there and the backend used when debugging people's info, so I'd like to see this come back in / be adjusted in some form rather than be removed